containers/ci-ubuntu-20.04/Dockerfile

```
@@ -0,0 +1,41 @@
+FROM ubuntu:20.04
```

This is basically the same as the 18.04 Dockerfile with a different base ubuntu and gcc version, if I'm not mistaken.
Would it make sense to have a single, parameterized Dockerfile for ubuntu?
e.g.:
```suggestion
ARG UBUNTU_VERSION=18.04
FROM ubuntu:${UBUNTU_VERSION}

ARG GCC_VERSION=8

```

and later on `apt install ... gcc-${GCC_VERSION} g++-${GCC_VERSION}`

---

Done